### PR TITLE
[IMP] hr_employee: employee creation notification

### PR DESCRIPTION
**Pull request by kgasior**
So far when new employee was created from applicant he got a mail with notification about employee creation. It's not a desired behaviour so this change makes just a log note about it.
There was also a  needed to make a small change in mailing - message_log_with_view method is no longer private as it needs to be used in this case.

task-3621545

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/models/mail_thread.py [hunk 1]
```diff
@@ -2199,7 +2199,7 @@ def message_notify(self, *,
         self._notify_thread(new_message, msg_values, **notif_kwargs)
         return new_message
 
-    def _message_log_with_view(self, views_or_xmlid, **kwargs):
```
Comment (sofiagvaladze) [inline addons/mail/models/mail_thread.py:2202]: @kgasior, why did you change the method signature? :thinking: 

First of all, makes sense that helper method is private, so, starts with` _`, but more importantly, we can't change method signature in stable. For more info - https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean 

So, change it back :smile:

**Comment by tivisse** [comment]
@robodoo r+
